src/Repository/ProductRepository.php

```diff
@@ -23,6 +23,7 @@ public function getProducts(int $page = 0, int $count = 3): iterable
         return $this->repository->createQueryBuilder('p')
             ->setMaxResults($count)
             ->setFirstResult($page * $count)
+            ->orderBy('p.createdAt', 'DESC')
```

Comment:
Introduce parameter `$orderBy = 'DESC'` instead of hard-coding it, so it can be used elsewhere for a different purposes also.